src/connectionhealthpolicy/ConnectionHealthData.ts

```diff
@@ -84,6 +87,9 @@ export default class ConnectionHealthData {
   setConsecutiveStatsWithNoPackets(stats: number): void {
     this.consecutiveStatsWithNoPackets = stats;
   }
+  setConsecutiveStatsWithNoAudioPacketsSent(stats: number): void {
```

Comment:
all these unnecessary setters for public members lol.

Reply:
Haha, but in my case this proved to be useful for unit testing though!